docs/devices/cloud-protocol/pgps.json

```diff
@@ -0,0 +1,6 @@
+{
+    "count": 42,
+    "interval": 240,
+    "day": 1516,
+    "time": 40655
```

Comment:
Please follow the format [defined in the firmware](https://github.com/nrfconnect/sdk-nrf/commit/e0ef12fe04d94265351d7536469eede3198533fc#diff-a5853d793834ff4df280f4c048833db4c4b862eef659ae9c68ffddef9fd0c5e3R125):

```json
{								
  "n":42,
  "int":240,	
  "day":15160,
  "time":40655
}
```